### PR TITLE
SE-17499: Clarification for Maven plugin usage in Studio.

### PR DESCRIPTION
**Pull request by fermujica**
Clarify that the Maven plugin config does not always apply to Studio run 
config.

Also added a link to runtime version config in Studio instructions.

### CODE REVIEW

**Review by egrave — DISMISSED** [review]

File: modules/ROOT/pages/munit-maven-plugin.adoc [hunk 1]
```diff
@@ -6,7 +6,9 @@ endif::[]
 :keywords: munit, testing, unit testing
 :page-aliases: munit-maven-support.adoc, to-set-up-munit-maven-plugin.adoc, faq-working-with-parent-pom.adoc, to-inherit-parent-plugin.adoc, to-ignore-parent-plugin.adoc, to-override-parent-plugin.adoc, munit-maven-plugin-configuration.adoc,to-configure-munit-maven-plugin-maven.adoc
 
-MUnit provides a Maven plugin that allows you to run your MUnit tests as part of your continuous integration environment.
+MUnit provides a Maven plugin that enables you to run your MUnit tests as part of your continuous integration environment.
+
+The following configurations to the `pom.xml` or `settings.xml` files of your Maven installation only apply when running Munit through Maven.
```
Comment (dukesphere) [inline modules/ROOT/pages/munit-maven-plugin.adoc]:
```suggestion
The following configurations to the `pom.xml` or `settings.xml` files of your Maven installation apply only when running MUnit through Maven.
```

File: modules/ROOT/pages/munit-maven-plugin.adoc [hunk 2]
```diff
@@ -813,8 +815,9 @@ Default value: `${project.build.directory}/test-mule/munit`
 
 | `runtimeVersion`
 | String
-| Version of the Mule runtime
-By default, this value is taken from the `minMuleVersion` parameter in the `mule-artifact.json` file.
+| Version of the Mule runtime +
+When running your tests from Maven, this value is taken from the `minMuleVersion` parameter in the `mule-artifact.json` file. +
```
Comment (dukesphere) [inline modules/ROOT/pages/munit-maven-plugin.adoc]:
```suggestion
When you run tests from Maven, this value is taken from the `minMuleVersion` parameter in the `mule-artifact.json` file. +
```

Comment (dukesphere) [reply modules/ROOT/pages/munit-maven-plugin.adoc]: otherwise it says that "this value" runs the test

**Review by dukesphere — COMMENTED** [review]
some small changes

**Review by dukesphere — APPROVED** [review]

**Pull request merged**